spec/support/fetchable_behavior.rb

```diff
@@ -200,10 +210,10 @@
 shared_examples_for 'a record with an indexable url' do
   describe 'validations' do
     context 'when the url extension is blacklisted' do
-      let(:movie_url) { "http://www.nps.gov/some.mov" }
+      let(:movie_url) { 'http://agency.gov/some.mov' }
       let(:record) { described_class.new(valid_attributes.merge(url: movie_url)) }
 
-      it "is not valid" do
+      it 'is not valid' do
         expect(record).not_to be_valid
         expect(record.errors.full_messages.first).to match(/extension is not one we index/i)
```

Comment:
Rubocop has flagged this as `Example has too many expectations`. We can discuss this. In the past, we've used multiple expectations pretty sparingly, and any egregious examples are generally caught in code review. My preference would be to turn off this cop, leaving it up to the discretion of the developer, with the understanding that _generally_ each spec should have a single expectation.

Reply:
Agreed. Modified RuboCop settings so this won't be flagged in future.